Resources/Audio/Backmen/Object/Devices/attributions.yml

```
@@ -0,0 +1,4 @@
+- files: ["experimentalsyndicateteleport.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "Taken from White Dream"
+  source: "https://github.com/frosty-dev/ss14-core/blob/master/Resources/Audio/White/Devices/expsyndicateteleport.ogg"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)